src/main/java/core/basesyntax/service/parser/ParserServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.parser;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int INDEX_FIRST = 0;
+    private static final int INDEX_SECOND = 1;
+    private static final int INDEX_THIRD = 2;
```

Comment:
it's better not to invent your own names
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int FRUIT_INDEX = 1;
    private static final int QUANTITY_INDEX = 2;
```